src/components/panels/VoidTraderPanel.vue

```diff
@@ -5,35 +5,25 @@
         <span class="pull-left">{{ locationLabel() }}</span>
         <TimeBadge :starttime="voidTrader.activation" :endtime="voidTrader.expiry" :interval="1000" />
       </b-list-group-item>
-      <b-list-group-item class="list-group-item-borderbottom" v-if="available()">
-        <Collapsible :headertext="`${voidTrader.character} ${$t('vt.inventory')}`">
-          <table class="table">
-            <thead>
-              <tr>
-                <th class="text-center col-xs-6">{{ $t('vt.item') }}</th>
-                <th class="text-center col-xs-2">{{ $t('currency.dabloons') }}</th>
-                <th class="text-center col-xs-4">{{ $t('currency.cred') }}</th>
-              </tr>
-            </thead>
-            <tbody>
-              <tr v-for="item in voidTrader.inventory" :key="`${item.item.replace(/\s/g, '').toLowerCase()}-inventory`">
-                <td>{{ item.item }}</td>
-                <td>{{ item.ducats }}</td>
-                <td>{{ item.credits }}</td>
-              </tr>
-            </tbody>
-          </table>
-        </Collapsible>
+      <b-list-group-item
+        class="list-group-item-borderbottom"
+        v-if="available()"
+        :items="this.items"
+        :fields="this.fields"
+      >
+        <b-table class="vt-table bounty-table" :items="this.items" :fields="this.fields" />
       </b-list-group-item>
     </b-list-group>
   </HubPanelWrap>
 </template>
 
-<style scoped>
-table.table td,
-table.table th,
-table.table > thead > tr {
-  border: 0;
+<style>
+.vt-table.table.b-table.bounty-table > tbody > tr > td {
```

Comment:
I think it might be time we rename this, instead of bounty-table, how about just item-table?